components/automate-ui/src/app/modules/configure/configure.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+import { LayoutFacadeService , Sidebar} from 'app/entities/layout/layout.facade';
+import { ConfigService } from 'app/services/config/config.service';
+@Component({
+  selector: 'app-configure',
+  templateUrl: './configure.component.html',
+  styleUrls: ['./configure.component.scss']
```

Comment:
This file configure.component.scss is empty. Can you check once if we actually need this file ?